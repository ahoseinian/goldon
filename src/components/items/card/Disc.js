import React from 'react'
import {Small} from '../../common/typography/Small'

const Disc = ({name, value, measure}) => (
  <div className="d-flex justify-content-between">
    <div>
      {name}:
    </div>
    <div>
      {value}
      <Small>{measure}</Small>
    </div>
  </div>
)

Disc.propTypes = {
  name: React.PropTypes.string.isRequired,
  value: React.PropTypes.string.isRequired,
  measure: React.PropTypes.string
}

export default Disc
