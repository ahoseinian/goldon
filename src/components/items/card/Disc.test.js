import React from 'react'
import {shallow} from 'enzyme'
import Disc from './Disc'
import toJson from 'enzyme-to-json'

describe('Disc', () => {
  it('renders', () => {
    const wrapper = shallow(<Disc name="name" value="value" />)
    expect(toJson(wrapper)).toMatchSnapshot();
  })
})
