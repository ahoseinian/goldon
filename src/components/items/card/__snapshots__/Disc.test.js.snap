exports[`Disc renders 1`] = `
<div
  className="d-flex justify-content-between">
  <div>
    name
    :
  </div>
  <div>
    value
    <styled.small />
  </div>
</div>
`;
